src/test/java/com/cdancy/bitbucket/rest/features/DefaultReviewersApiLiveTest.java

```diff
@@ -48,6 +53,67 @@ public void testListDefaultReviewersOnNewRepo() {
         assertThat(conditionList).isEmpty();
     }
 
+
+    @Test(dependsOnMethods = {"testListDefaultReviewersOnNewRepo"})
+    public void testCreateCondition() {
+        Long requiredApprover = 1L;
+        Matcher matcherSrc = Matcher.create(Matcher.MatcherId.ANY, true);
+        Matcher matcherDst = Matcher.create(Matcher.MatcherId.ANY, true);
+        List<User> listUser = new ArrayList<>();
+        listUser.add(getDefaultUser());
+        CreateCondition condition = CreateCondition.create(null, generatedTestContents.repository, matcherSrc,
+                matcherDst, listUser, requiredApprover);
+
+        Condition returnCondition = api().createCondition(projectKey, repoKey, condition);
+        conditionId = returnCondition.id();
+        validCondition(returnCondition, requiredApprover, Matcher.MatcherId.ANY_REF, Matcher.MatcherId.ANY_REF);
+    }
+
+    @Test(dependsOnMethods = {"testListDefaultReviewersOnNewRepo", "testCreateCondition"})
+    public void testCreateConditionOnError() {
+        Long requiredApprover = 1L;
+        Matcher matcherSrc = Matcher.create(Matcher.MatcherId.ANY, true);
+        Matcher matcherDst = Matcher.create(Matcher.MatcherId.ANY, true);
+        List<User> listUser = new ArrayList<>();
+        listUser.add(getDefaultUser());
+        CreateCondition condition = CreateCondition.create(null, generatedTestContents.repository, matcherSrc,
+                matcherDst, listUser, requiredApprover);
+
+        Condition returnCondition = api().createCondition(projectKey, "1234", condition);
+        assertThat(returnCondition.errors()).isNotEmpty();
+        assertThat(returnCondition.targetRefMatcher()).isNull();
+        assertThat(returnCondition.sourceRefMatcher()).isNull();
+        assertThat(returnCondition.reviewers()).isNull();
+        assertThat(returnCondition.repository()).isNull();
+    }
+
+    @Test(dependsOnMethods = {"testListDefaultReviewersOnNewRepo"})
+    public void testCreateConditionMatcherDifferent() {
+        Long requiredApprover = 1L;
+        Matcher matcherSrc = Matcher.create(Matcher.MatcherId.MASTER, true);
+        Matcher matcherDst = Matcher.create(Matcher.MatcherId.DEVELOPMENT, true);
+        List<User> listUser = new ArrayList<>();
+        listUser.add(getDefaultUser());
+        CreateCondition condition = CreateCondition.create(null, generatedTestContents.repository, matcherSrc,
+                matcherDst, listUser, requiredApprover);
+
+        Condition returnCondition = api().createCondition(projectKey, repoKey, condition);
+        validCondition(returnCondition, requiredApprover, Matcher.MatcherId.MASTER, Matcher.MatcherId.DEVELOPMENT);
+    }
+
+    @Test(dependsOnMethods = {"testListDefaultReviewersOnNewRepo", "testCreateCondition", "testCreateConditionMatcherDifferent"})
```

Comment:
No, I need to depend on method testCreateCondition and testCreateConditionMatcherDifferent because testListConditions do check if It have 2 conditions return by api. I will remove third.